src/index.html

```diff
@@ -22,6 +22,413 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="Company name logo Moyo"
```

Comment:
It's good practice to provide a more descriptive `href` for the logo link, pointing to the homepage usually, instead of using `#`.